programs/compile/compile-with-webpack.js

```diff
@@ -26,21 +26,26 @@ function createCompiler(config) {
   }
 }
 
-function modifyForProduction(config) {
+function modifyForProduction(config, args) {
   config.devtool = "cheap-module-source-map"
   config.plugins.push(
     new webpack.optimize.OccurrenceOrderPlugin(),
     new webpack.optimize.DedupePlugin(),
-    new webpack.optimize.UglifyJsPlugin({
-      compress: { warnings: false },
-      comments: false,
-      minimize: true
-    }),
     new webpack.DefinePlugin({
       "process.env.NODE_ENV": JSON.stringify("production")
     })
   )
 
+  if (!args.includes("--no-uglify")) {
```

Comment:
We want `uglify` to be the default behavior? 
```javascript
if(`args.includes("--uglify")`)
```
would be easier to read otherwise :)